.github/workflows/ci.js.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow will do a clean installation of node dependencies, build the source code and run tests
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Run tests
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
```

Comment:
Question: We don't need to cache the results, right?

Reply:
Don't _need_ to , but would be nice. I'll see if I can figure it out.

Reply:
from [59 seconds](https://github.com/ftrackhq/javascript-api/runs/7745846913?check_suite_focus=true) to [9 seconds](https://github.com/ftrackhq/javascript-api/runs/7745893126?check_suite_focus=true) - nice